pages/admin/reuses/new.vue

```diff
@@ -110,11 +114,25 @@ onMounted(async () => {
   }
 })
 
-const datasets = useState<Array<Dataset | DatasetV2 | DatasetSuggest>>(
+const datasets = useState<Array<DatasetV2 | DatasetSuggest>>(
   REUSE_DATASET_STATE,
   () => [],
 )
 
+onMounted(async () => {
+  if (!route.query.dataservice_id) return
```

Comment:
What is the use case for this query param ?

Reply:
To be able to have a link on the dataservice page to create a reuse for a specific dataservice (not yet there)